extensions/wikia/EditPageLayout/js/plugins/MiniEditor.js

```diff
@@ -75,7 +75,7 @@
 		},
 
 		editorActivated: function(event) {
-
+		
 			$(window).on('beforeunload.PreventLeaveBeforeSave', function (e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "e". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[project-id]: wikia:app